core/src/serve_repair.rs

```diff
@@ -151,7 +151,8 @@ impl RequestResponse for AncestorHashesRepairType {
 #[derive(Default)]
 struct ServeRepairStats {
     total_requests: usize,
-    dropped_requests: usize,
+    dropped_requests_bandwidth: usize,
+    dropped_requests_batching: usize,
```

Comment:
`_batching` is a bit confusing to me.
maybe `_load_shed' or `_rate_limit`.

also `_outbound_bandwidth` to be more explicit.